mon-pix/translations/en.json

```
@@ -1575,7 +1575,7 @@
         "e-learning": "E-Learning",
         "hybrid-training": "Hybrid training",
         "in-person-training": "In person training",
-        "modulix": "Modulix",
+        "modulix": "Module Pix",
```

Est-ce qu'on corrige les trads aussi?

```suggestion
        "modulix": "Pix Module",
```

---

ah oui bien vu !